.github/workflows/self-nightly-past-ci-caller.yml

```diff
@@ -0,0 +1,140 @@
+name: Self-hosted runner (nightly-past-ci-caller)
+
+on:
+  push:
+    branches:
+      - run_nightly_ci*
+      - run_past_ci*

```

Comment:
I will need to measure how long one round of workflow run will take, and add a schedule even here with proper interval.